src/test/java/org/jenkinsci/plugins/scriptsecurity/sandbox/whitelists/StaticWhitelistTest.java

```diff
@@ -122,6 +122,8 @@ static void sanity(URL definition) throws Exception {
             new MethodSignature(String.class, "isEmpty"),
             // Does not exist until Java 15.
             new MethodSignature(CharSequence.class, "isEmpty"),
+            // Overrides DefaultGroovyMethods.stripIndent(String) in Java 15+.
+            new MethodSignature(String.class, "stripIndent"),
```

Comment:
We are using "override" here to refer specifically to Java method overriding, which is not quite what is happening in this case, so I would just note that the new method does not exist until Java 15+ like `CharSequence.isEmpty()` instead.

```suggestion
            // Do not exist until Java 15.
            new MethodSignature(CharSequence.class, "isEmpty"),
            new MethodSignature(String.class, "stripIndent"),
```

Reply:
Done